src/websock/websock.c

```diff
@@ -412,13 +412,8 @@ static void http_resp_handler(int err, const struct http_msg *msg, void *arg)
 	/* here we are ok */
 
 	conn->state = OPEN;
-	conn->tc = mem_ref(http_req_tcp(conn->req));
-	conn->sc = mem_ref(http_req_tls(conn->req));
 	(void)tcp_conn_peer_get(conn->tc, &conn->peer);
```

Comment:
looks like `conn->tc` is accessed here, is the conn->tc NULL?

Reply:
No, conn->tc is set in the conn_handler(), which is called just before the response handler.